lxd/resources/storage.go

```diff
@@ -20,6 +21,28 @@ var devDiskByID = "/dev/disk/by-id"
 var runUdevData = "/run/udev/data"
 var sysClassBlock = "/sys/class/block"
 
```

Comment:
Go prefers camelCase over snake_case, so decodeUdev or something along those lines would be better. It should also live in utils.go rather than in storage.go